Makefile

```diff
@@ -420,11 +423,13 @@ endif
 	@echo "Ensuring git repository is clean..."
 	$(E)$(MAKE) git-clean-check
 
-lint: lint-code
+lint: lint-code lint-md
 
 lint-code: $(golangci_lint_bin)
 	$(E)PATH="$(go_bin_dir):$(PATH)" GOLANGCI_LINT_CACHE="$(golangci_lint_cache)" $(golangci_lint_bin) run ./...
 
+lint-md:
+	$(E)docker run -v "$(DIR):/workdir" $(markdown_lint_image) "**/*.md"
```

Comment:
Is this the only spot where we require docker like this? Would it be better to install it into go_bin_dir instead?

Reply:
that was my initial idea, but if we do that we requires to have rubygems (or node.js if we choose another library).
With that we'll need to require to have on linux or windows instances just for building (or we'll need to install similar we do with golang)
But depending on docker makes it easier and we can use images they provide.